habitat-baselines/habitat_baselines/config/multi_agent/plan_pop.yaml

```diff
@@ -0,0 +1,139 @@
+# @package _global_
```

Comment:
Ideally this could derive from the `pop_play` config, but I couldn't get Hydra to correctly override the `/habitat_baselines/rl/policy@habitat_baselines.rl.policy.agent_1: hl_neural_multi_agent` entry. If I used the `override` keyword, running training would fail due to Hydra error. If I don't use the `override` keyword, the Hydra tests fail.